appveyor.yml

```diff
@@ -44,7 +44,12 @@ before_build:
 
 build_script:
   - ps: C:\projects\jmespath-net\bin\update-version.ps1 -Version $Env:APPVEYOR_BUILD_VERSION
-  - dotnet build "src\jmespath.net" /p:BuildInParallel=false -c %CONFIGURATION% --version-suffix %LABEL%
+  - ps: dotnet build "src\jmespath.net" /p:BuildInParallel=false -c %CONFIGURATION% --version-suffix %LABEL%

```

Comment:
I think this change break the build and need to be change to work with PS command using $Env:

Reply:
✔